webrender/src/render_backend.rs

```diff
@@ -213,6 +213,7 @@ impl Document {
             &self.output_pipelines,
             &self.frame_builder_config,
             &mut self.current.scene,
+            0,
```

Comment:
why are we passing 0 here?

Reply:
because the value isn't necessary on the sync path, and also we plan to remove this path altogether soon, right?

Reply:
please put a comment then saying that this is not used because the picture IDs get reset. Frankly, it seems easier to just put the correct value in here in the first place

Reply:
there is a comment elsewhere, where the purpose of this value is explained, but you're right, i'll look at piping it through properly. i did this when i was just figuring out how everything piped together